node/task.ts

```diff
@@ -117,7 +117,7 @@ export function getVariables(): VariableInfo[] {
  * @param     secret  whether variable is secret.  optional, defaults to false
  * @returns   void
  */
-export function setVariable(name: string, val: string, secret: boolean = false): void {
+export function setVariable(name: string, val: string, secret: boolean = false, isOutput:boolean = false): void {
```

Comment:
Is output controls if data is send to server. https://github.com/Microsoft/vsts-agent/blob/45c0d943ce6474a08ef9865384bd5be5f0a459d7/src/Agent.Worker/ExecutionContext.cs#L215